.github/workflows/cd.yml

```diff
@@ -0,0 +1,60 @@
+name: CD
```

Comment:
By default this workflow should not run, right? How is that going to be controlled? I guess it's still WIP